### PR TITLE
Bug 1134884 — Log current roomConnectionId for GET and POST /rooms/:token.

### PR DESCRIPTION
**Pull request by Natim**
https://bugzilla.mozilla.org/show_bug.cgi?id=1134884

### CODE REVIEW

File: loop/routes/rooms.js [hunk 1]
```diff
@@ -414,21 +415,30 @@ module.exports = function (apiRouter, conf, logError, storage, auth,
                 sendError(res, 410, errors.EXPIRED, "Participation has expired.");
                 return;
               }
-              res.status(200).json({
-                expires: ttl
+              storage.getRoomParticipant(req.token, participantHmac, function(err, participant) {
+                if (res.serverError(err)) return;
+                req.roomConnectionId = participant.id;
```
Comment (leplatrem) [inline loop/routes/rooms.js]: A small comment expliciting why this is id is replaced by another would be useful (I couldn't follow)

Comment (Natim) [inline loop/routes/rooms.js]: It is not replaced, it is set.

Comment (leplatrem) [inline loop/routes/rooms.js]: You set a few lines above though

Comment (Natim) [inline loop/routes/rooms.js]: Yes but for another router (handleJoin versus handleRefresh) The behavior is routed on the action payload.

**Comment by ametaireau** [comment]
r+

**Pull request merged**